mmeds/server.py

```diff
@@ -669,6 +672,17 @@ def process_data(self, public=False, **kwargs):
             datafiles = self.load_data_files(otu_table=kwargs['otu_table'])
             reads_type = None
             barcodes_type = None
+        elif cp.session['upload_type'] == 'lefse':
+            datafiles = self.load_data_files(lefse_table=kwargs['lefse_table'])
+            if 'subclass' in kwargs.items():
+                reads_type = 'subclass'
+                if 'subjects' in kwargs.items():
+                    reads_type = reads_type + '_subjects'
+            elif 'subjects' in kwargs.items():
```

Comment:
I think this should be `x in kwargs.keys()`. `.items()` returns `(key, value)` pairs so `in` would try to match against that